plugins/panels/similarity_search/operators.py

```
@@ -335,7 +334,7 @@ def _embed_query_image(ctx):
         return model.embed(img)
 
 
-class InitSimilarityRunOperator(foo.Operator):
+class InitSimilarityRunOperator(foo.Operator):  #
```

Ruff (0.15.10)

[warning] Line with empty comment

Delete the empty comment

(PLR2044)